tests/console/http_srv.pm

```diff
@@ -40,13 +60,22 @@ sub run {
     );
 }
 
+sub run {
+    select_console 'root-console';
+    script_run("df -h > /dev/$serialdev", 0);
+    check_apache_install();
+    enable_apache_config();
+    start_apache_service();
+    check_apache_service();
+    check_apache_funtion();
```

Comment:
I wonder how clean is this, as the more services we have to check, the more lines will be added here, which makes things a bit poorly scalable...

Reply:
> s/funtion/function (5 ocurrences)

done.

Reply:
> I wonder how clean is this, as the more services we have to check, the more lines will be added here, which makes things a bit poorly scalable...

will try function pointer call as call_func().